frontend/src/components/visualizations/Visualization.tsx

```diff
@@ -95,21 +117,16 @@ export const Visualization = (props: previewProps) => {
 								// if no visualization config exist, guess which widget to use by looking at the mime type of
 								// the raw bytes
 								else {
-									//check if file size is greater than threshold, if it is, then show no visualization
-									if (fileSummary && fileSummary.bytes && fileSummary.bytes >= config["rawDataVisualizationThreshold"]) {
-										console.log("File is greater than threshold");
-									}
-									// try match mime type first
-									// then fallback to match main type
-									// to instantiate components correspondingly
-									else if (
-										fileSummary &&
+									//to make sure file size is less than threshold
+									if (
+										fileSummary && fileSummary.bytes && fileSummary.bytes < config["rawDataVisualizationThreshold"] &&
 										fileSummary.content_type !== undefined &&
 										((fileSummary.content_type.content_type !== undefined &&
 											visComponentDefinition.mimeTypes.includes(
 												fileSummary.content_type.content_type
 											)) ||
-											(fileSummary.content_type.content_type === undefined) && (fileSummary.content_type.main_type !== undefined &&
```

Comment:
use this logic the same way in the useEffect above to check all the visComponentDefinitions for a matching mime type - only if we don't have a component to show the raw bytes show the "No visu"